aruba/features/config/step_definitions/my_steps.rb

```diff
@@ -18,4 +18,11 @@
           #{ALL_ENDPOINTS_CONFIG}
           """
         }
+end
+
+When(/^I enroll random certificate \-config "([^"]*)" \-profile (.*) with (.+)?$/) do |config, profile, flags|
+  cn = " -cn " + random_cn
+  cmd = "vcert enroll -config #{config} -profile #{profile} #{cn} #{flags}"
+
+  steps %{Then I try to run `#{cmd}`}
 end
```

Comment:
Missing EOF here

Reply:
This is not introduced by my changes. But sure, will fix it by next commit

Reply:
Done by latest changes